storage/src/main/java/tech/pegasys/teku/storage/server/DepositStorage.java

```diff
@@ -16,6 +16,7 @@
 import com.google.common.base.Supplier;
```

Comment:
Switch `java.util.function.Supplier` instead of the google version.  It's only the package that needs to change.
```suggestion
import java.util.function.Supplier;
```

Reply:
what about the Suppliers? use that from google still?

Reply:
Yep that still comes from the google package.